### PR TITLE
Work for assignment 11

### PR DESCRIPTION
**Pull request by barkls**

### CODE REVIEW

File: infer_structcol/main.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        return self['wavelength'].values
+    @property
+    def meas(self):
+        return self['measurement'].values
+    @property
+    def sigma(self):
+        return self['sigma'].values
+
+class Sample:
+    def __init__(self, particlesize, thickness, particleindex, matrixindex, mediumindex=1, incidentangle=0):
```
Comment (victoriahwang89) [inline infer_structcol/main.py]: Maybe underscores would be better to define the variable names: particle_size, etc

File: infer_structcol/main.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        self.particlesize = particlesize # can we do something clever here with units? maybe using pint?
```
Comment (victoriahwang89) [inline infer_structcol/main.py]: Yeah, I think the user should define the variable particlesize as a pint variable, so the class should expect pint variables

File: infer_structcol/main.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        self.thickness = thickness # again with the units
+        self.matrixindex = matrixindex
+        self.mediumindex = mediumindex
+        self.incidentangle = incidentangle #should default be 90? pi/2?
```
Comment (victoriahwang89) [inline infer_structcol/main.py]: If the incident light is collimated, then it should be set to 0 to follow the coordinate system in the montecarlo code. This is also how the scattering angle is defined.

File: infer_structcol/main.py (hunk 2 above)
Comment (barkls) [inline infer_structcol/main.py]: I'm not sure what a pint variable looks like, so we might need to address this later.

**Pull request merged**